pkg/vcs/testos.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package vcs
+
+import (
+	"io"
+)
+
+type Testos struct {
+	*git
+}
+
+func NewTestos(dir string) *Testos {
+	return &Testos{
+		git: newGit(dir, nil),
+	}
+}
+
+func (ctx *Testos) Bisect(bad, good string, trace io.Writer, pred func() (BisectResult, error)) ([]*Commit, error) {
+	commits, err := ctx.git.Bisect(bad, good, trace, pred)
+	return commits, err
+}
+func (ctx *Testos) PreviousReleaseTags(commit string) ([]string, error) {
+	return ctx.git.previousReleaseTags(commit, false)
+}
+func (ctx *Testos) EnvForCommit(commit string, kernelConfig []byte) (*BisectEnv, error) {
+	return &BisectEnv{
+		Compiler: "/usr",
```

Comment:
Nobody should use this "/usr" compiler, right? The test builder does not actually invoke it as far as I see. If so, it will be better to name it along the lines of "test-compiler-dont-use". Then any attempt to invoke it will lead to here. Seeing invocation of "/usr" may be confusing.

Reply:
build.CompilerIdentity() seems to be the one user of this:
```
	output, err := osutil.RunCmd(time.Minute, "", compiler, arg)
	if err != nil {
		return "", err
	}
```
Should we mock it?

Reply:
Ah, I see. I will think about this.